js/client/inspector.js

```diff
@@ -0,0 +1,92 @@
+/* jshint -W051, -W020 */
+/* global global:true, window, require */
```

Comment:
why should this need `window`?

Reply:
windows is not necessary anymore. was in middle and was forgotten.